unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/NFTIcons/NFTTypeIconsAndColors.cs

```diff
@@ -9,23 +9,23 @@ public class NFTTypeIconsAndColors : ScriptableObject
 
     public Sprite GetTypeImage(string nftType)
     {
-        for(int i=0; i< nftIcons.Length; i++)
+        foreach (var t in nftIcons)
```

Comment:
Try to avoid to use "only 1 letter" variables and lets use more descriptive names for them. I think it will improve the code readability:
```suggestion
        foreach (var icon in nftIcons)
```